tests/test_docs.py

```diff
@@ -1,11 +1,19 @@
 import platform
+from pathlib import Path
 
 import pytest
 from pytest_examples import CodeExample, EvalExample, find_examples
 
+root_dir = Path(__file__).parent.parent
+
+examples = find_examples(
+    str(root_dir / 'dirty_equals'),
```

Comment:
Without changing this file, I couldn't just click a button to run tests in PyCharm.

The conversion to `str` is because that's what the `find_examples` type annotation expects and my IDE was warning me about it, although a path object works fine. I think the `find_examples` annotation should be `Union[Path, str]`.

Reply:
fine by me, happy to update `pytest-examples`.